src/utils/helper.js

```diff
@@ -1,11 +1,24 @@
 import fs from 'fs';
 import path from 'path';
+import toml from 'toml';
+
+function readAndParseTomlConfig(filePath) {
+    try {
+        const content = fs.readFileSync(filePath, 'utf-8');
+        return toml.parse(content);
+    } catch (err) {
+        console.error('Error reading or parsing TOML file:', err);
```

Comment:
When an invalid toml file is passed, it generates output like this:
<img width="1233" alt="image" src="https://github.com/sdthaker/Node-TILify/assets/71615163/6cde0258-4a59-4d29-9c63-222a105adaee">

I personally dont think all of this information is necessary to show to the user because it is of no use to them and could lead to ambiguity. Simply remove the `err` variable at the end of the log and just show the text "Error reading or parsing TOML file".

Reply:
Resolved in commit [8371c9eb](https://github.com/sdthaker/Node-TILify/pull/13/commits/21441eaecd095e242fb3dc277e10194bf512c3e2)